pyriemann/utils/covariance.py

```diff
@@ -60,14 +109,6 @@ def _sch(X):
     Notes
     -----
     .. versionadded:: 0.3
-
-    References
-    ----------
-    .. [1] `A shrinkage approach to large-scale covariance estimation and
-        implications for functional genomics
-        <https://pubmed.ncbi.nlm.nih.gov/16646851/>`_
-        J. Schafer, and K. Strimmer. Statistical Applications in Genetics and
-        Molecular Biology, Volume 4, Issue 1, November 2005.
```

Comment:
I would have kept the ref here. Otherwise it's maybe less clear where to find the info.

Reply:
Ok, but since these functions are private, there are not displayed in API, and thus users can't see the references.
We can duplicate references. What do you think?

Reply:
yes duplicate them it does not harm.